packages/tx/test/eip4844.spec.ts

```diff
@@ -387,16 +387,16 @@ tape('Network wrapper deserialization test', async (t) => {
       chainId: '0x1',
       accessList: [],
       maxFeePerDataGas: '0x5f5e100',
-      versionedHashes: ['0x01317228841f747eac2b4987a0225753a4f81688b31b21192ad2d2a3f5d252c5'],
+      versionedHashes: ['0x01a638b488178191c51eb548b89ce6e144f897b250b8cba834a7e4accd1cf297'],
```

Comment:
Hi @g11tech, just having another look at this PR for release notes. Out of interest: why is the changeset for this PR and this tx tests actually so small? 🤔

Wouldn't e.g. the following from above (just totally abitrary example) also change and therefore a lot of hash and the like comparisons?

```typescript
t.equal(
  bytesToHex(unsignedTx.getMessageToSign(true)),
  '8ce8c3544ca173c0e8dd0e86319d4ebfe649e15a730137a6659ba3a721a9ff8b',
  'produced the correct transaction hash'
)
```

Just wondering what "I am not getting" here. 🙂

Reply:
@holgerd77  the txMeta (see at L392) changes exactly what you been mentioning

Reply:
👍